### PR TITLE
change text for placeholder of metric chooser

### PR DESCRIPTION
**Pull request by Shivan-997**
# change text for placeholder of metric chooser

closes: #2508

### CODE REVIEW

**Comment by ce-bo** [comment]
A quick Note in the Changelog for this improvement Would be nice

File: visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 				class="metric-search"
 				ng-model="$ctrl._viewModel.searchTerm"
 				type="text"
-				placeholder="Metric (highest value)"
+				placeholder="Edge Metric (highest value)"
```
Comment (BridgeAR) [inline visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html:23]: I wonder what the `highest value` actually stands for. Might that be a legacy leftover?

**Review by BridgeAR — APPROVED** [review]
LGTM. I just wonder if we might want to remove the highest value part as well?

File: visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html (hunk 1 above)
Comment (ce-bo) [reply visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html:23]: It is a Kind of legend. In the metrics List you can See the highest value for mcc for instance in brackets.

Comment (ce-bo) [reply visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html:23]: Let‘s Remove it

Comment (BridgeAR) [reply visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html:23]: Good point. I just let someone have a look at it and they told me it's better to have the description than removing it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 -   Fix delta values of secondary metrics not shown in delta mode within attribute side bar [#2539](https://github.com/MaibornWolff/codecharta/issues/2539).
 -   Use icon tag instead of font awesome icon [#2537](https://github.com/MaibornWolff/codecharta/pull/2537).
+-   Rename text for placeholder of metric chooser [#2508](https://github.com/MaibornWolff/codecharta/pull/2547)
```
Comment (ce-bo) [inline CHANGELOG.md:14]:
```suggestion
-   Rename text for placeholder of metric chooser [#2547](https://github.com/MaibornWolff/codecharta/pull/2547)
```

**Review by ce-bo — APPROVED** [review]

**Pull request merged**